src/Connections/GitlabRepo.ts

```diff
@@ -351,6 +351,30 @@ export class GitLabRepoConnection extends CommandConnection<GitLabRepoConnection
         });
     }
 
+    @botCommand("create-confidential", "Create a confidental issue for this repo", ["title"], ["description", "labels"], true)
```

Comment:
This repeats a lot of code from the `create()` above, can we refactor it a little?

Reply:
I've deduped, but honestly it doesn't feel worth splitting out any further than this to me?

Reply:
Fair, alright :)